src/styles/blocks/container.scss

```diff
@@ -0,0 +1,8 @@
+.container {
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(4, 200px);
+  row-gap: 46px;
+  column-gap: 48px;
+  padding: 50px 40px 72px;
+}
```

Comment:
If you are using grid - please implement as it is described in the [task](https://github.com/mate-academy/layout_catalog/blob/master/grid.md)